C-Sharp/VectorDistanceCalculation/PE11VectorDistanceCalculation/Point2DCollection.cs

```diff
@@ -17,7 +19,8 @@ class Point2DCollection
         private Random random = new Random();
         public Point2DCollection()
         {
-            for (int i = 0; i < collectionSize;)
+            //Don't do this type of loop
```

Comment:
I didn't want the int i to increment if it encountered a point that is the same as the one being added.  I guess a While loop may be preferred in this case?

Reply:
hmmm. not a fan but I see your point (ha).

Reply:
What would be your approach?